static/bidder-params/gumgum.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "GumGum Adapter Params",
+  "description": "A schema which validates params accepted by the GumGum adapter",
+  "type": "object",
+  "properties": {
+    "zone": {
+      "type": "string",
+      "description": "A tracking id used to identify GumGum zone.",
+      "minLength": 8
+    },
+    "deviceHeight": {
```

Comment:
Hey, out of curiosity: why do you use `deviceWidth` and `deviceHeight` as bidder params here?

OpenRTB already defines these as `request.device.w` and `request.device.h`... so you wouldn't need anything bidder-specific to capture them.

Also: the device size will vary from user to user. These bidder params are defined by Publishers define for a particular page. How would you expect them to fill these out?

Reply:
Sure I can clarify. I was testing my adapter with my coworker who created a test page running the Prebid client and we noticed that the device object never got sent in the bid request. We spoke with a publisher that wants to call us through Prebid server and asked if they can send over the device info in the request and they told us that they do not control what gets sent over in the request since they run Prebid.js. So I was under the impression that Prebid.js is not sending the device info in the request. 

Regarding how the publisher would set this up, the way my coworker defined the device width and height params was by using the functions window.innerWidth() and window.innerHeight(). 

Please let me know if there was perhaps some issue with our test page setup and the device info does actually get sent over through Prebid.js.

Reply:
Aha! Yeah, that makes sense. Thanks!

Your test page was probably fine... Prebid.js just isn't sending these _yet_. I opened https://github.com/prebid/Prebid.js/pull/3336 to add them, though.

Feel free to test using that branch, if you want to stay ahead of the PRs & merge ^^. I haven't worked with Prebid.js in a long time, but I believe it should fix your problem.

Reply:
Gotcha! Will you be merging your changes soon? I’m out of office this week so I can test with your branch on Monday. If I still don’t see the device info included in the request, how should we proceed?

Reply:
ugh... so: yes, I plan to merge these changes as soon as possible. They're small, so I can nag the Prebid.js team for a review if necessary.

...but right now the tests don't pass, and I can't even build it locally because of https://github.com/prebid/Prebid.js/issues/3330. The Prebid.js folks are waiting on https://github.com/AveVlad/gulp-connect/pull/259, which is outside of our control.

So... I definitely will try to get this in quickly, but can't promise a timeline either.

Reply:
Ah I see. I know you said it’s hard to give a timeline but do you think the issues will be resolved by the end of December? Trying to merge my adapter before the end of Q4 :) Also in the case you think it won’t be merged by the end of December, would it be terrible to merge my adapter as is and get rid of the device width and height in the ext object once your changes have been merged and the device info is being sent over?

Reply:
We merged it now :). They're actually releasing today too, so these changes should be live in v1.35.0 in the next few hours. If the tests fail during release validation, it might take a few days... but shouldn't be any longer than that.

Reply:
Awesome news! Okay I will go ahead and remove the device width and height from the extension object and see how it goes. Thanks!